File: Programming-languages-and-laboratory/Java/Exercises/05_compito_java_2015_07_15/Stampante.java
import java.util.*;

public class Stampante extends Prodotto {
    private int peso;

    public Stampante(int codice, String modello, String produttore, int giorno, int mese, int anno, int prezzo, int peso) {
        super(codice, modello, produttore, giorno, mese, anno, prezzo);
        this.peso = peso;
    }

    public String toString(){
        return "Stampante\t" + codice + "\t" + modello + "\t" + produttore + " \t" + giorno + "\t" + mese + "\t" + anno + "\t" + prezzo + "\t-\t" + peso;
    }
}

File: Programming-languages-and-laboratory/Python/Exercises/05_compito_java_20150715/Stampante.py
from Prodotto import Prodotto


class Stampante(Prodotto):
    def __init__(self, tipo, codice, modello, produttore, giorno, mese, anno, prezzo, peso):
        super().__init__(tipo, codice, modello, produttore, giorno, mese, anno, prezzo)
        self._peso = peso

    def __str__(self):
        return "Stampante\t" + super().__str__() + "\t-\t" + str(self._peso)
